proto/sourcehub/orbis/ring.proto

```
@@ -37,7 +48,7 @@ message Ring {
   string policy_id = 10;
   repeated RingConfirmation confirmations = 11;
   UpgradeInfo upgrade_info = 12 [(gogoproto.nullable) = false];
-  DemeritConfig demerit_config = 13 [(gogoproto.nullable) = false];
+  ReportingConfig reporting = 13 [(gogoproto.nullable) = false];
```

Buf (1.71.0)

[error] cannot find `gogoproto.nullable` in this scope

(COMPILE)